webapp/app/forms/steps/lotse/personal_data.py

```diff
@@ -62,7 +62,8 @@ class InputForm(SteuerlotseBaseForm):
                 ('TH', _l('form.lotse.field_bundesland_th'))
             ],
             render_kw={'data_label': _l('form.lotse.field_bundesland.data_label'),
-                       'input_req_err_msg': _l('form.lotse.field_bundesland_required')}
+                       'input_req_err_msg': _l('form.lotse.field_bundesland_required')},
+            validators=[InputRequired(_l('form.lotse.steuernummer.input_required'))], 
```

Comment:
We now have two conflicting messages for the Bundesland because the `input_req_err_msg` is used for the input required, isn't it?

Reply:
Otherwise, we should get rid of it and probably everything else that is connected to the `input_req_err_msg`

Reply:
Done :) should be fine